CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ This is the documentation for [https://docs.npmjs.com/](https://docs.npmjs.com/)
 
 First, `npm install` the dependencies. This will install Gatsby, et al.
 
-Next, `npm run develop` to start the test server to view your changes. The Gatsby server will be started on port 8000. You can navigate to `http://localhost:8000` to view the site live.
+Next, `npm run develop` to start the test server to view your changes. The Gatsby server will be started on port 8000. You can navigate to `http://localhost:8000` to view the site live. If you're on Windows, change the syntax for setting environment variables in the `package.json` file based on your terminal before running this command. The syntax for command prompt is `set ENV_VAR=value & command`.
```

Comment:
If this file gets changed, folks are almost certainly going to accidentally commit those changes, which we don't want.

We may just have to say that local development on windows doesn't work with the shortcut scripts, and that you have to manually run the associated commands.

Reply:
Makes sense! Should I amend the existing commit or add a new commit?

Reply:
You can keep adding commits, we can squash the PR when it's ready.